src/tasks.md

```diff
@@ -0,0 +1,263 @@
+
+*As a policy author*
+
+- *end-to-end testing of your policy*: test your policy against crafted Kubernetes requests, and ensure your policy behaves as you expect. You can even test context-aware policies, that require access to a running cluster
+- *Embed metadata in your Wasm module*: the binary is annotated with the permissions it needs to execute
+- *Publish policies to OCI registries*: the binary is a fully compliant OCI object and can be stored in OCI registries
+
+*As a cluster administrator*
+
+- *Inspect remote policies*: given a policy in an OCI registry, or in an HTTP server, show all static information about the policy.
+- *Dry-run of a policy in your cluster*: test the policy against crafted Kubernetes requests, and ensure the policy behaves as you expect given the input data you provide. You can even test context-aware policies, that require access to a running cluster, also in a dry-run mode.
+- *Generate initial `ClusterAdmissionPolicy` scaffolding for your policy*: generates a `YAML` file with all the required settings, which can be applied to your Kubernetes cluster using `kubectl`
+
+#### Installation
+
+`kwctl` binaries, for the stable releases, are directly available from the [GitHub repository](https://github.com/kubewarden/kwctl/releases).
+
+> **NOTE**: If you want to build `kwctl` from the development branch, you need to install [Rust](https://www.rust-lang.org/tools/install). And for building `kwctl`, please refer to the `Build kwctl from source` section in the [GitHub repo](https://github.com/kubewarden/kwctl).
+
+#### Usage
+
+As stated above, `kwctl` will allow you to perform an end-to-end testing of the policies.
+
+You can list all the `kwctl` options and subcommandes by running the following command:
+
+```shell
+$ kwctl --help
+kwctl 0.2.4
+Flavio Castelli <fcastelli@suse.com>:Rafael Fernández López <rfernandezlopez@suse.com>
+Tool to manage Kubewarden policies
+
+USAGE:
+    kwctl [FLAGS] <SUBCOMMAND>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+    -v               Increase verbosity
+
+SUBCOMMANDS:
+    annotate       Add Kubewarden metadata to a WebAssembly module
+    completions    Generate shell completions
+    help           Prints this message or the help of the given subcommand(s)
+    inspect        Inspect Kubewarden policy
+    manifest       Scaffold a Kubernetes resource
+    policies       Lists all downloaded policies
+    pull           Pulls a Kubewarden policy from a given URI
+    push           Pushes a Kubewarden policy to an OCI registry
+    rm             Removes a Kubewarden policy from the store
+    run            Runs a Kubewarden policy from a given URI
+    verify         Verify a Kubewarden policy from a given URI using Sigstore
+```
+
+Here are few examples of the commands you should run, depending on the task you want to perform:
+
+- *List the policies*: lists all the policies stored in the local `kwctl` registry
```

Comment:
Just a note: I think it's fine to keep words lowercase after colons in cases like this, wherein it begins an incomplete phrase. But you could cap these also for consistency- up to you!